### PR TITLE
add cp_build to start building from jenkins

### PR DESCRIPTION
**Pull request by elbetasal**

### CODE REVIEW

File: cp_build.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+tools:
+  jdk: 11.0.5-zulu11.35.15
+  maven: 3.6.1
+test:
+  stages:
+    unit:
+      script: mvn verify -X
+  services:
+    testing:
+      branch: master
```
Comment (eric-swanson) [inline cp_build.yml]: Deleting these three lines will fix the `fit listsuites` errors

File: cp_build.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+tools:
+  jdk: 11.0.5-zulu11.35.15
+  maven: 3.6.1
```
Comment (eric-swanson) [inline cp_build.yml]: Is there a reason not to use the latest jdk (11.0.6) and maven (3.6.2 I think)?

File: cp_build.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+test:
+  stages:
+    unit:
+      script: mvn verify -X
```
Comment (eric-swanson) [inline cp_build.yml]: Please add a `junit` field for good hygiene

File: cp_build.yml (hunk 2 above)
Comment (elbetasal) [reply cp_build.yml]: I copy/paste from other project but I pushed the latest in 09dc561

File: cp_build.yml (hunk 3 above)
Comment (elbetasal) [reply cp_build.yml]: Done in 09dc561

File: cp_build.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+      junit: "target/surefire-reports/*.xml"
+publish:
+  tag:
+    library: java/maven
+build:
+  script: mvn clean verify -Pcheckstyle
```
Comment (eric-swanson) [inline cp_build.yml]: This is running tests during the build phase

It's not terribly important for this project, but not an example we'd want others to follow

`-DskipTests` ?

Comment (elbetasal) [reply cp_build.yml]: That makes sense.

**Pull request merged**